DS-Session-1/dataSructure/LinkedListImplementationOfStack.java

```diff
@@ -0,0 +1,81 @@
+package dataSructure;
+
+/**
+ * this class is used to create a node for the linked list
+ * @author Ishita_Mathur
+ *
+ */
+class StackNode
```

Comment:
Class `StackNode` is almost same for Stack & Queue. A separate class can be used at both implementation.